File: GlassDoor/ClientApp/src/app/models/currency.ts
import { JobDetails } from "./job-details";

export class Currency {

  constructor(id?: number, symbol?: string, name?: string, code?: string,
    jobDetails?: JobDetails) {

    this.id = id;
    this.name = name;
    this.symbol = symbol;
    this.code = code;
    this.jobDetails = jobDetails;

  }
  public id: number;
  public name: string;
  public symbol: string;
  public code: string;
  public jobDetails: JobDetails;
}

File: DAL/Models/Currency.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Models
{
    public class Currency
    {
        public int Id { get; set; }
        public string Symbol{ get; set; }

        public string Name { get; set; }

        public string Code { get; set; }

        public ICollection<JobDetails> JobDetails { get; set; }
    }
}
